packages/frontend-shared/src/gql-components/HeaderBarContent.vue

```diff
@@ -299,7 +300,20 @@ const prompts = sortBy([
   },
 ], 'interval')
 const isForceOpenAllowed = ref(true)
+const isOpenDelayElapsed = ref(false)
+
+onMounted(() => {
+  setTimeout(() => isOpenDelayElapsed.value = true, 2000)
```

Comment:
nice, this makes sense of the `tick` in the test code for mounting.

Reply:
Sidenote: we could simplify a little and remove the mounted hook updating the ref with `useTimeout` (https://vueuse.org/shared/usetimeout/)